sentences/sv/light_HassTurnOn.yaml

```diff
@@ -8,9 +8,14 @@ intents:
           - "<slå_på>  <area> [<alla>] <ljuskällor>"
           - "<slå_på> [<alla>] <area>[s]<ljuskällor>"
         slots:
-          domain: "light"
+          domain: light
 
       - sentences:
           - "<slå_på> <ljuskällor> <name>"
         slots:
           domain: light
+
+      - sentences:
+          - "<slå_på> <name> <i_på> <area>"
+        slots:
+          domain: light
```

Comment:
I don' think that this should be under the `domain: light` since there isn't anything defining that this is just for lights in the sentence. 
This could also trigger `starta diskmaskinen i köket` that wouldn't be a light but the dishwasher.

Reply:
makes sense, can I just remove domain altogether or how would you solve it? if you know...

Reply:
yes it is for light but the sentence does not limit to light it would also match a sentence like:
`turn on the tv in the livingroom` = "starta tv i vardagsrummet` but in that case it would include the domain light. I'm not sure how that should be handled. 
I think that this sentence should be moved from light to `homeassistant_HassTurnOn.yaml` and `homeassistant_HassTurnOff.yaml`

Reply:
maybe we should add to `homeassistant_HassTurnOn.yaml` and even other files like `fan_HassTurnOn.yaml` but that doesn't mean we should remove it from `light_HassTurnOn.yaml`, right?

Reply:
I don't know how same sentences would work with different Intents, adding the same to homeassistant_HassTurnOn would trigger 2 intents?